trunk/SUAVE/Vehicle.py

```diff
@@ -378,3 +380,137 @@ def get_children(self):
         """       
         
         return [Vehicle_Mass_Properties]
+
+class Constraints(Data):
+    """ Constraints class for the constraint analysis
+
+    Assumptions:
+    None
+    
+    Source:
+    None
+    """    
+    def __defaults__(self):
+        """This sets the default values.
+    
+            Assumptions:
+            None
+    
+            Source:
+            N/A
+    
+            Inputs:
+            None
+    
+            Outputs:
+            None
+    
+            Properties Used:
+            None
+            """         
+            
+        self.tag                = 'Constraint analysis'
+        self.plot_tag           = False
+
+
+        # Defines default constraint analyses
```

Comment:
vehicle.analyses breaks a fundamental assumption of SUAVE having data/analysis orthogonality

Reply:
Unfortunately, I did not get the issue here. Could you explain it a bit more?